utils/scripts/ocm/ocm.py

```diff
@@ -267,34 +294,119 @@ def install_rhods(self):
+            if ret is None:
+                print("Failed to add ldap identity provider")
+                sys.exit(1)
+
+    def delete_idp(self, idp_name):
+        """Deletes Identity Provider"""
+
+        cmd = ("ocm delete idp -c {} {}".format(self.cluster_name,
+                                                idp_name))
         ret = execute_command(cmd)
         if ret is None:
-            print("Failed to add identity provider of "
-                  "type {}".format(self.idp_type))
-            sys.exit(1)
+            print("Failed to delete identity provider of "
+                  "type {}".format(idp_name))
 
-    def add_user_to_group(self, group="cluster-admins"):
+    def add_user_to_group(self, user="", group="cluster-admins"):
         """Adds user to given group"""
 
-        cmd = ("ocm create user {} --cluster {} "
-               "--group={}".format(self.htpasswd_cluster_admin,
-                                   self.cluster_name, group))
+        if user == "":
+            user = self.htpasswd_cluster_admin
+
+        if ((group == "rhods-admins") or
+           (group == "rhods-users") or
+           (group == "rhods-noaccess")):
+            cmd = "oc adm groups add-users {} {}".format(group, user)
+        else:
+            cmd = ("ocm create user {} --cluster {} "
+                   "--group={}".format(user,
+                                       self.cluster_name, group))
         ret = execute_command(cmd)
         if ret is None:
             print("Failed to add user {} to group "
-                  "{}".format(self.htpasswd_cluster_admin, group))
+                  "{}".format(user, group))
             sys.exit(1)
 
+        # Logging users/groups details after adding
+        # given user to group
+
+        cmd = "oc get users"
+        users_list = execute_command(cmd)
+        print ("Users present in cluster: {}".format(users_list)) 
+
+        cmd = "oc get groups"
+        groups_list = execute_command(cmd)
+        print ("Groups present in cluster: {}".format(groups_list))
+
+    def create_group(self, group_name):
+        """Creates new group"""
+
+        cmd = "oc adm groups new {}".format(group_name)
+        ret = execute_command(cmd)
+        if ret is None:
+            print("Failed to add group "
+                  "{}".format(group_name))
+            sys.exit(1)
+
+    def add_users_to_rhods_group(self):
+        """Add users to rhods group"""
+
+        # Adds user ldap-admin1..ldap-admin20
+        for i in range(1, 21):
+            self.add_user_to_group(user="ldap-admin"+str(i),
+                                   group="rhods-admins")
+
+        # Adds user ldap-user1..ldap-user20
+        for i in range(1, 21):
+            self.add_user_to_group(user="ldap-users"+str(i),
+                                   group="rhods-users")
+
+        self.create_group("rhods-noaccess")
+        # Adds user ldap-noaccess1..ldap-noaccess20
+        for i in range(1, 21):
```

Comment:
I would recommend using a argument variable for the end of the range (`21`) to allow the ability change it dynamically

Reply:
comments incorporated.